src/main/scala/rpsGame.scala

```diff
@@ -1,69 +1,65 @@
-    println(playerMove(player))
-    println(aiPicker(playerMove(player)))
-    println(winText(checker(playerMove(player), aiPicker(playerMove(player)))))
-    println()
-    println()
+    println("Player: "+playerMove(player))
+    aiPicker(playerMove(player))
+    println("AI: " + move)
+    wins = checker(playerMove(player), move)
+    if (wins == "P") pWin+=1
+    if (wins == "A") aWin+=1
+    message = winText(wins)
+    println(message + "\n Score:\n  Player: " + pWin + "\n  AI: " + aWin)
   }
 }
 
-  def winText(result: String): String ={
-    var pCounter: Int = 0
-    var aCounter: Int = 0
-    var message: String = ""
-    if (result == "P")
-      pCounter+=1
-      message = "Player wins! \n Score: \nPlayer " + pCounter + "\nAI " + aCounter
-    if (result == "A")
-      aCounter+=1
-      message = "AI wins! \n Score: \nPlayer " + pCounter + "\nAI " + aCounter
-    if (result == "D")message = "Draw! \n Score: \nPlayer " + pCounter + "\nAI " + aCounter
-    if (result == "W") message = "Stop trying to break the game, R, P or S please!!!! \nScore: \n Player " + pCounter + "\n AI " + aCounter
-
-    message
+  def winText(result: String): String = result match{
+    case ("P") => "Player wins!"
+    case ("A") => "AI wins!"
+    case ("D") => "It's a draw! :("
+    case ("W") => "Stop trying to break the game, R P or S please!!"
   }
 
   def checker(pMove: String, aMove: String): String = (pMove, aMove) match{
-    case (_,_) if pMove == "Rock" && aMove == "Scissors" || pMove == "Paper" && aMove == "Rock" || pMove == "Scissors" && aMove == "Paper" => "P"
-    case (_,_) if aMove == "Rock" && pMove == "Scissors" || aMove == "Paper" && pMove == "Rock" || aMove == "Scissors" && pMove == "Paper" => "A"
-    case (_,_) if pMove == aMove => "D"
-    case (_,_) if pMove == "WRONG CHARACTER" => "W"
+    case (p,a) if (p == "Rock" && a == "Scissors") || (p == "Paper" && a == "Rock") || (p == "Scissors" && a == "Paper") => "P"
+    case (p,a) if (a == "Rock" && p == "Scissors") || (a == "Paper" && p == "Rock") || (a == "Scissors" && p == "Paper") => "A"
+    case (p,a) if p == a => "D"
+    case (p,_) if p == "WRONG CHARACTER" => "W"
   }
 
-  def playerMove(move: String): String = move match {
-    case "r" => "Rock"
-    case "p" => "Paper"
-    case "s" => "Scissors"
+  def playerMove(move: String): String = move.charAt(0) match {
+    case 'r' => "Rock"
+    case 'p' => "Paper"
+    case 's' => "Scissors"
     case _ => "WRONG CHARACTER"
   }
 
   def aiPicker(pMove: String): String = {
-
-    var move: String = ""
-    var first: Boolean = true
-    while (first) {
+    while (firstMove) {
       import scala.util.Random
       val A = Array("Rock", "Paper", "Scissors")
       move = Random.shuffle(A.toList).head
-      first = false
+      firstMove = false
+      move
     }
-    while (first) {
-      var pRCounter: Int = 0
-      var pPCounter: Int = 0
-      var pSCounter: Int = 0
     if (pMove == "Rock") pRCounter += 1
```

Comment:
instead of 3 if checks you could make a call to a helper method that would do a pattern matching on the pMove